scan.c

```diff
@@ -29,6 +33,14 @@ struct iio_scan_context {
 	bool scan_local;
 };
 
+#ifdef WITH_MATLAB_BINDINGS_API
+struct iio_scan_block {
```

Comment:
Put 'struct iio_scan_block' right before 'iio_scan_block_scan' in the #ifdef and I'm happy with the commit.